### DISCUSSION TITLE
Fast API Status code

### DISCUSSION THREAD

**Discussion by ajsalkp** [fastapi/fastapi#9944] [Questions]
How to rename status code 400 as Invalid Request instead of Bad Request?

**Comment by Kludex** [comment]
On which case? You mean on the server logs?

**Reply by ajsalkp** [reply]
Yes.For server logs

**Comment by YuriiMotov** [comment]
This is not related to FastAPI but to the ASGI server you are using.
And I believe there is no easy way to change it
